.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,10 @@
+# Contributing to GSRI Zeus Manager
+
+## Bug reports
+If you spot an issue with any of the Manager's functions, feel free to open an issue on our Github repository. Please provide every needed information required to reproduce the issue, so we can diagnose it appropriately.
+
+## New features
+You are free to fork this project and modify it so it can better suits your needs. If you feel like your work can benefit the whole community, feel free to open a pull-request, and we will be glad to study your modifications !
```

Comment:
Add infos : fork and PR shall be based on latest master, and people should PR towards our master branch